request/src/main/java/com/vimeo/networking2/VimeoService.kt

```diff
@@ -81,33 +81,33 @@ internal interface VimeoService {
         @Path(TYPE) type: ConnectedAppType
     ): VimeoCall<Unit>
 
-    @GET
+    @GET("{path}")
     fun getPublishJob(
         @Header(AUTHORIZATION) authorization: String,
-        @Url uri: String,
+        @Path("path", encoded = true) uri: String,
```

Comment:
Optional: you could change the corresponding parameter names in this file from `uri` to `path` as well for clarity.

Reply:
Going to leave it as "URI" already didn't match "URL" is named so to match the vimeo API's usage of the word "URI" rather than retrofit's.